dockercompose/types.go

```diff
@@ -0,0 +1,67 @@
+package dockercompose
+
+type ImageType int64
+
+const (
+	Play ImageType = iota
+	Dev
+	Contribute
+)
+
+func (it ImageType) String() string {
+	switch it {
+	case Play:
+		return "play"
+	case Dev:
+		return "dev"
+	case Contribute:
+		return "contribute"
```

Comment:
i think essentials and flex is missing?

Reply:
aaa sry, thats the image type for coding.
could you rename it, its slightly confusing with the real image types.
maybe "DevPurposeType" or sth like that?